proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -141,8 +141,9 @@ service IAMService {
     option (common.v1.permissions) = "write:roles";
     // custom account-id expression
     option (qdrant.cloud.common.v1.account_id_expression) = "role.account_id";
-    // custom supported-actor-types expression. (At this moment only users can create roles)
+    // custom supported-actor-types expression. (Users and management keys can create roles)
     option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_USER;
+    option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_MANAGEMENT_KEY;
```

Comment:
what do you think about removing them? in all the other methods, when we don't have specific requirements we just don't set the supported_actor_types and we use the default values.

Reply:
done.